modules/features/discover/src/main/java/au/com/shiftyjelly/pocketcasts/discover/view/CategoriesBottomSheetAdapter.kt

```diff
@@ -0,0 +1,36 @@
+package au.com.shiftyjelly.pocketcasts.discover.view
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.LinearLayout
+import android.widget.TextView
+import androidx.recyclerview.widget.ListAdapter
+import androidx.recyclerview.widget.RecyclerView
+import au.com.shiftyjelly.pocketcasts.discover.R
+import au.com.shiftyjelly.pocketcasts.servers.model.DiscoverCategory
+import au.com.shiftyjelly.pocketcasts.servers.model.NetworkLoadableList
+import coil.load
+
+class CategoriesBottomSheetAdapter(val onCategoryClick: (NetworkLoadableList) -> Unit) : ListAdapter<DiscoverCategory, CategoriesBottomSheetAdapter.CategoryViewHolder>(CATEGORY_DIFF) {
+    class CategoryViewHolder(val itemView: View) : RecyclerView.ViewHolder(itemView) {
```

Comment:
No need to keep a reference to `itemView`. `RecyclerView.ViewHolder` has this property on its own.